docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/google-tag-manager-custom-template/v3-tags/ecommerce-tag-template/ecommerce-tag-configuration/index.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Ecommerce Tag Configuration"
+sidebar_position: 100
+---
+
+```mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+```
+
+## Ecommerce API
+
+Use the native Snowplow Ecommerce API or transitional GA4/UA ecommerce adapter APIs for existing dataLayer implementations using those formats. To get full value from the Snowplow Ecommerce plugin we recommend using the native API when possible.
```

Comment:
Links to `/docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/plugins/snowplow-ecommerce/index.md` and `/docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/plugins/snowplow-ecommerce/index.md#ga4ua-ecommerce-transitional-api`